integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/RefreshCarbonTableCommand.scala

```diff
@@ -91,6 +95,11 @@ case class RefreshCarbonTableCommand(
           registerAggregates(databaseName, dataMapSchemaList)(sparkSession)
         }
         registerTableWithHive(databaseName, tableName, tableInfo)(sparkSession)
+        // Register partitions to hive metastore in case of hive partitioning carbon table
+        if (tableInfo.getFactTable.getPartitionInfo != null &&
+            tableInfo.getFactTable.getPartitionInfo.getPartitionType == PartitionType.NATIVE_HIVE) {
+          registerAllPartitionsToHive(tablePath, databaseName, tableName, sparkSession)
```

Comment:
I think passing AbsolutionTableIdentifier is better, it has all information. It is created in line 65

Reply:
ok